VenafiPS/Public/Remove-TppEnginesAssignedToFolder.ps1

```diff
@@ -0,0 +1,152 @@
+<#
```

Comment:
I'd combine this with the other function and call it Remove-TppEngineFolder since they both do the same thing.  You could also make engine optional and just providing a folder path removes all engines.

Reply:
implemented